leading-workshops/add-folks-to-2i2c-github-teams.qmd

```diff
@@ -46,44 +80,144 @@ This is what the invitation looks like in the GitHub website interface - clickin
+
+We use a dedicated GitHub Organization - [nasa-openscapes-workshops](https://github.com/nasa-openscapes-workshops) - to manage access, with [GitHub Teams](https://github.com/orgs/nasa-openscapes-workshops/teams) for workshops and Champions Cohorts.
+
+### 1. Create a team in [nasa-openscapes-workshops](https://github.com/orgs/nasa-openscapes-workshops/teams)
+
+There are several teams in this organization; the `AdminTeam` team is for 
+members who have permission to create teams and add members to teams.
+
+- If this is for a one-off workshop, name the team `[workshop-name]-[workshop-date]`, with 
+workshop date in the format `yyyy-mm-dd`
+- If this is for a new champions cohort, name the team `nasa-champions-yyyy`
+
+### 2. Add team name to workshop registry
+
+We keep a registry of all workshops in [this sheet](https://docs.google.com/spreadsheets/d/1Sg4fMDGxMLI0p5cgK5xyamFJpKC6NyNneqXLDReSKcg/edit#gid=695033382). Add the name 
+of the GitHub team to `2i2c access type` column in the appropriate row.
+
+### 3. Add teams to 2i2c Hub configuration
+
+Make a pull request to the Openscapes [common.values.yaml](https://github.com/2i2c-org/infrastructure/blob/main/config/clusters/openscapes/common.values.yaml) file in 
+the 2i2c infrastructure repository, adding the new team to the `allowed-teams` 
+item in whichever sections are appropriate for your workshop/cohort:
+
+- `display_name: Python`
+- `display_name: R`
+- `display_name: Matlab`
+- `display_name: "Bring your own image"`
+
+These will determine which images are available to participants when they log in 
+to the Hub.
+
+Team specification is in the form `[github-org]:[team]` (e.g., `nasa-openscapes-workshops:champions-access-2i2c-2024`)
+
+You also need to add the team to the `allowed_organizations` item under 
+`hub > config > GitHubOAuthenticator`
+
+You can see an example pull request [here](https://github.com/2i2c-org/infrastructure/pull/3944).
+
+### 4. Share Google Form with participants needing access for the cohort or workshop
+
+Ask users: Please add your GitHub Username to this [Google form](https://forms.gle/htV2PxJwRaP8jDpC6) 
+for NASA Openscapes JupyterHub Access.
+
+### 5. Add users to a Team in a batch using [kyber](https://github.com/Openscapes/kyber)
+
+::: {.callout-tip}
+Do this fairly close to the day of the workshop to try to get the greatest 
+number particpants to have filled out the form, and reduce the number of manual 
+additions on the day of the workshop.
+:::
+
+1. Open the [spreadsheet of form responses](https://docs.google.com/spreadsheets/d/1TgokoQa-jYncYSOjeYoWYRuDAbyo4uUR-MSzUotvM00/edit?resourcekey#gid=1416760746), select the GitHub usernames from the rows containing the people you want to add, and copy it to your clipboard.
+
+::: {.callout-tip}
+Some workshop instructors will use their own form to record participants,
+rather than the Openscapes form. In that case use those responses instead.
```

Comment:
```suggestion
rather than the Openscapes form. In that case use those responses instead. But do paste all those rows into this Main form; it is important that we have a record of everyone using the Hub and a way to contact them via email.
```